infinite/index.ts

```diff
@@ -20,7 +18,7 @@ import {
   SWRInfiniteResponse,
   MutatorCallback,
   Middleware
-} from '../src/types'
+} from 'swr'
```

Comment:
Even if we fixed those types, the dependency `import defaultConfig from '../src/utils/config'` will still import types directly from `'../src/types'`. So I feel this is a tricky problem.

Reply:
Good catch. Seems the types of whole swr core are emitted into package. Need to find a way to get rid of it.

Reply:
Only include the source files in subpath modules can avoid emitting swr types. But still fews left like `with-middlewares`. We can improve it later by later.

```
infinite/dist/
├── index.esm.js
├── index.js
├── infinite
│   └── index.d.ts
└── src
    └── utils
        ├── env.d.ts
        ├── hash.d.ts
        ├── helper.d.ts
        ├── normalize-args.d.ts
        ├── serialize.d.ts
        └── with-middleware.d.ts

3 directories, 9 files
```